src/types.ts

```diff
@@ -56,6 +56,16 @@ export interface ISoundBody {
   backgroundTrackId: string;
 }
 
+export interface ISoundTemplatesFilteringBody {
+  /** Placeholder allowed parameters to be updated after database is completed */
```

Comment:
jsdoc in wrong place. lets wait for the db

Reply:
done